.github/workflows/bench_trigger.yml

```diff
@@ -22,42 +22,51 @@ jobs:
         id: setup_pr
         if: contains(github.event.pull_request.labels.*.name, 'benchmark PR')
         env:
+          AUTHOR: ${{ github.event.pull_request.assignee.login }}
           BRANCH: ${{ github.head_ref }}
+          PIPE_TRIGGER_TOKEN: ${{ secrets.BENCH_PIPE_TRIGGER }}
+          SHA: ${{ github.event.pull_request.head.sha }}
+          PR_NUMBER: ${{ github.event.pull_request.number }}
         run: |
           SHORT_SHA=$(git rev-parse --short ${{ github.event.pull_request.head.sha }})
           curl -s -X POST \
-            --fail \
-            -F token=${{ secrets.BENCH_PIPE_TRIGGER }} \
+            --fail-with-body \
+            -F "token=$PIPE_TRIGGER_TOKEN" \
             -F "ref=main" \
-            -F "variables[SHA]=${{ github.event.pull_request.head.sha }}" \
+            -F "variables[SHA]=$SHA" \
             -F "variables[SHORT_SHA]=${SHORT_SHA}" \
             -F "variables[BRANCH]=$BRANCH" \
-            -F "variables[PR]=${{ github.event.pull_request.number }}" \
-            -F "variables[AUTHOR]=${{ github.event.pull_request.assignee.login }}" \
+            -F "variables[PR]=$PR_NUMBER" \
+            -F "variables[AUTHOR]=${AUTHOR:-heat_team}" \
```

Comment:
What does `:-heat_team` do?

Reply:
Default value, in case the author field is empty.